src/mainboard/pcengines/apu1/bootorder_def

```diff
@@ -7,10 +7,9 @@
 /pci@i0cf8/*@11/drive@0/disk@0
 /pci@i0cf8/*@11/drive@1/disk@0
 /pci@i0cf8/*@11/drive@2/disk@0
-/rom@genroms/pxe.rom
```

Comment:
@piotr-kleins there should be an entry here too

Reply:
Added entry and slimed apu3 bootorder file